crates/currency/src/amount.rs

```diff
@@ -224,6 +224,11 @@ mod math {
             Ok(if self.le(other)? { self.clone() } else { other.clone() })
         }
 
+        pub fn max(&self, other: &Self) -> Result<Self, DispatchError> {
```

Comment:
This function can be removed now

Reply:
I chose to leave it since it's a potentially useful utility function